GameData/RealismOverhaul/Engine_Configs/LMAE_Config.cfg

```diff
@@ -1,101 +1,116 @@
-//  ==================================================
-//  Lunar Module Ascent Engine (LMAE)
-
-//  Gross Mass: 95 kg
-//  Throttle Range: N/A
-//  Burn Time: 465 s
-//  O/F Ratio: 1.6
-
-//  Sources:
-//      http://www.alternatewars.com/BBOW/Space/Reference_Spacecraft_Engines.htm
-//      https://www.hq.nasa.gov/alsj/LM10HandbookVol1.pdf
-//      http://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19700026405.pdf
-//      http://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/20100027314.pdf
-
-//  Used by:
-//      AIES
-//      FASA
-//      Squad
-//  ==================================================
-
-@PART[*]:HAS[#engineType[LMAE]]:FOR[RealismOverhaulEngines]
-{
-	%title = Lunar Module Ascent Engine (LMAE)
-	%manufacturer = Bell / Rocketdyne
-	%description = Pressure-fed engine used for the ascent module of the Apollo lunar lander.
-  @mass = 0.0816
```

Comment:
If you hide whitespace changes, this becomes readable at least.

Why did you remove the patch to mass?

Reply:
The patch to mass does nothing, as MEC will overwrite any mass changes to its own settings.

Reply:
Alright, then.  @Capkirk123 , you good with the rest of the patch?